frontend/src/components/TE/TaskDefinition.vue

```diff
@@ -3,11 +3,15 @@
     <template v-for="(chunk, index) in item.chunks">
       <v-row class="wrap pb-1" :key="`${item['key']}-${index}-layout`">
         <v-col v-bind="layout.link">
-          <div>
+          <v-badge v-if="item.isRestored" color="#2196F3">
+            <template v-slot:badge>SEd</template>
             <a :href="getLink(chunk.fileName)">
               {{ chunk.fileName }}
             </a>
-          </div>
+          </v-badge>
+          <a v-else :href="getLink(chunk.fileName)">
+            {{ chunk.fileName }}
+          </a>
```

Comment:
Is it possible to avoid duplication of `a` tag?

Also, as I've mentioned in the card just now, the badge should be added to the task ID, not the source file name.

Reply:
Without something in the badge slot a small blue circle is present.

Reply:
Then let it be always badge, but in case `restored is false` put `Non-SEd` into the badge.